sport/app/rugby/conf/RugbyLifecycle.scala

```diff
@@ -34,6 +34,17 @@ trait RugbyLifecycle extends GlobalSettings with ExecutionContexts {
       RugbyStatsJob.fetchPastScoreEvents
     }
 
+    Jobs.deschedule("LiveMatchesStat")
+    Jobs.schedule("LiveMatchesStat", "0 * * * * ?") {
+      RugbyStatsJob.fetchLiveMatchesStat
+    }
+
+    Jobs.deschedule("PastMatchesStat")
+    Jobs.schedule("PastMatchesStat", "0 0/30 * * * ?") {
+      RugbyStatsJob.fetchPastMatchesStat
+    }
```

Comment:
Just been going through a bunch of schedulers on Admin and they all kick off on the minute and I'm changing them to space out. Might be worth doing here so for example changing `"0 * * * * ?` to `"10 * * * * ?` will still run every minute just 10 seconds after the minute rather than on the minute. Might not be worth it here but easier to debug etc.